Taskfile.yml

```diff
@@ -57,6 +57,10 @@ tasks:
       COGITO_TEST_REPO_OWNER: '{{default "pix4d" .COGITO_TEST_REPO_OWNER}}'
       COGITO_TEST_GCHAT_HOOK:
         sh: 'echo {{default "$(gopass show cogito/test_gchat_webhook)" .COGITO_TEST_GCHAT_HOOK}}'
+      COGITO_TEST_GH_APP_CLIENT_ID: '{{default "Iv23lir9pyQlqmweDPbz" .COGITO_TEST_GH_APP_CLIENT_ID}}'
+      COGITO_TEST_GH_APP_INSTALLATION_ID: '{{default "64650729" .COGITO_TEST_GH_APP_INSTALLATION_ID}}'
```

Comment:
So the client ID would be always the same, representing `cogito` itself, and the installation ID would need to be created by everybody using the feature of GH app authentication ?

Reply:
are the defaults just for CI? I would not provide them at all

Reply:
@aliculPix4D ?

Reply:
![Screenshot from 2025-04-29 09-56-34](https://github.com/user-attachments/assets/503ba2d8-826f-4772-9fc6-be4c55048232)

Do you think this description is good enough? Otherwise, can you open: https://github.com/apps/pci-cogito-test and configure the description accordingly... I am not the GitHub organization admin... @odormond created the app for me... 

I do plan to update the contributing docs in this repo, mentioning the existence of this test app (next PR), but its still important that app description in GitHub UI is written correctly.

Reply:
This is good enough, thanks.